packages/api/package.json

```diff
@@ -22,12 +22,13 @@
     "faker": "^5.5.3",
     "googleapis": "^59.0.0",
     "jsonwebtoken": "^8.5.1",
+    "nodejs": "*",
```

Comment:
Do we need this?

Reply:
This has been removed from the latest commit.